index.html

```diff
@@ -0,0 +1,191 @@
+</label>
+</div>
+
+
+</div>
+
+
+
+<div class="contact-information">
+    <label class="containerradio" for="name">Name </label>
+<input type="text" name="name" id="firstname" placeholder="Name" required>
+</div>
+    
+<div class="contact-information">
+    <label for="lastname">Lastname </label>
+<input type="text" name="lastname" id="lastname" placeholder="Lastname" required>
+</div>
+
+<div class="contact-information">
+<label for="phonenumber">Phonenumber</label>
+<input type="tel" name="phonenumber" id="phone" placeholder="Phonenumber" required>
+</div>
+
+<div class="contact-information">
+<label for="mail">Mail</label>
+<input type="email" name="mail" id="mail" placeholder="Your-email@here.com" required>
+</div>
+
+<div class="contact-information">
+<label for="username">Username</label>
+<input type="text" name="username" id="username" placeholder="Username" required>
+</div>
+
+<div class="contact-information">
+<label for="pass">Password: </label>
+<input type="password" id="password" name="password" minlength="6" placeholder="6 characters minimum" required>
+</div>
+
+<div class="contact-information">
+<label for="over-20-years">I comfirm that I'm 20 years or older</label>
+<input type="checkbox" id="over-20-years" name="over-20-years" required>
+</div>
+
+<div class="contact-information">
+<label class="checkbox-label" for="newsletter">subscribe to our newsletter</label>
+<input type="checkbox" id="newsletter" name="newsletter">
+</div>
+
+<input type="submit" value="save">
+
+
+</fieldset>
+  </form>
+   </div> 
+   
+
+    
+  
+
+  </section>
+    <section class="accordion-div">
+      
+      <div class="accordion-form">
+      <div class="top-according">
+      <h1>FAQ</h1>
+    </div>
+      <div class="accordion">Full festival ticket</div>
+      <div class="answer">
+        <p>Ticket for the entire festival. The days with full music schedules 
+          are the last four days of the festival, from 30 June till 3 July..</p>
+      </div>
+
+      <div class="accordion">One-day ticket</div>
+      <div class="answer">
+        <p>You have the opportunity to buy One-Day tickets to the following days; 
+          Friday 2 July and Saturday 3 July. Every One-Day ticket/wristband is valid 
+          from 8am until 10am the next day.</p>
+      </div>
+      <div class="accordion">Name on the ticket?</div>
+      <div class="answer">
+        <p>The name on the ticket does not restrict the ticket to that person. 
+          So don't worry if a friend bought your ticket or you received it as a present. 
+          You can still use it.</p>
+      </div>
+      <div class="accordion">Buying tickets at the entrance</div>
+      <div class="answer">
+        <p>If tickets aren’t sold out before the festival begins, 
+          you can buy them at the entrance at a slightly higher price.</p>
+      </div>
+    
+    </section>
+
+ 
+
+<footer class="footer-div">
```

Comment:
Your code is great but you may have a look at the structure of it to make it even more easy to read. :) In VS code I think there is a shortcut to format it (Shift+Option+F on a Mac).